index.js

```diff
@@ -1,7 +1,5 @@
 'use strict';
 
-var toString = {}.toString;
-
 module.exports = Array.isArray || function (arr) {
-  return toString.call(arr) === '[object Array]';
+  return arr && arr.constructor === Array;
```

Comment:
This is not viable; .constructor isn’t a reliable property and Array would only be === if it came from the same realm.